EvalError({
  name: "EvalError",
  message: "something went wrong",
  stack: "EvalError: something went wrong\n    at Object.<anonymous> (/Users/ayon/git/fron/test/errors/index.js:10:17)\n    at Module._compile (internal/modules/cjs/loader.js:955:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)\n    at Module.load (internal/modules/cjs/loader.js:811:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:723:14)\n    at Module.require (internal/modules/cjs/loader.js:848:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (/Users/ayon/git/fron/test/get-types.js:13:5)\n    at Module._compile (internal/modules/cjs/loader.js:955:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)\n    at Module.load (internal/modules/cjs/loader.js:811:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:723:14)\n    at Module.require (internal/modules/cjs/loader.js:848:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at /Users/ayon/git/fron/node_modules/mocha/lib/mocha.js:250:27\n    at Array.forEach (<anonymous>)\n    at Mocha.loadFiles (/Users/ayon/git/fron/node_modules/mocha/lib/mocha.js:247:14)\n    at Mocha.run (/Users/ayon/git/fron/node_modules/mocha/lib/mocha.js:576:10)\n    at Object.<anonymous> (/Users/ayon/git/fron/node_modules/mocha/bin/_mocha:637:18)\n    at Module._compile (internal/modules/cjs/loader.js:955:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)\n    at Module.load (internal/modules/cjs/loader.js:811:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:723:14)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)\n    at internal/main/run_main_module.js:17:11"
})